File: IOSwiftUI/Backoffice.UI/src/screens/members/models/BOMemberListRequestModel.ts
import BaseRequestModel from "../../../common/models/BaseRequestModel";

class BOMemberListRequestModel extends BaseRequestModel {

    count: number;
    start: number;

    constructor() {
        super();

        this.count = 0;
        this.start = 0;
    }
}

export default BOMemberListRequestModel;

File: IOSwiftUI/Common/Messages/Members/BOMemberListRequestModel.cs
using System;
using System.ComponentModel.DataAnnotations;
using IOSwiftUI.Common.Messages.Base;

namespace IOSwiftUI.Common;

public class BOMemberListRequestModel : RequestModel
{
    [Required]
    public int Count { get; set; }

    [Required]
    public int Start { get; set; }
}
